Formula/geesefs.rb

```diff
@@ -0,0 +1,18 @@
+class Geesefs < Formula
+  desc "Finally, a good FUSE FS implementation over S3"
+  homepage "https://github.com/yandex-cloud/geesefs"
+  url "https://github.com/yandex-cloud/geesefs/archive/refs/tags/v0.30.5.tar.gz"
+  sha256 "149ca98073641b8c051a361950a7111520be1da032489614d8fdcb1316543bc6"
+
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", "#{bin}/geesefs"
+  end
+
+  test do
+    system "#{bin}/geesefs", "--version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Ok, geesefs has tests, but they need small adjustments for macos because they use some standard gnu command line tools

Reply:
We don't need to run upstream tests. You just need to simulate basic user interaction with geesefs. See the other `test do` blocks in this repository for details. (Some of them still do `--help` or `--version`, but those will just need to be fixed.)